index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css"/>
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="">
+            <a href="" class="logo">Alexis NIYO</a>
+        <input class="menu-btn" type="checkbox" id="menu-btn" />
+        <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
+        <ul class="menu">
+          <li><a href="#"></a></li>
+          <li><a href="#"></a></li>
+          <li><a href="#"></a></li>
+          <li><a href="#"></a></li>
+          <li><a href="#"></a></li>
+          <li><a href="#"></a></li>
+          <li><a href="#"></a></li>
+          <li><a href="#"></a></li>
+        </ul>
+        </nav>
+    </header>
+
+    <section class="headline">
+        <h1 class="heading-1">Hey There. I’m Alexis</h1> 
+        <h2 class="heading-2">I am Software Developer</h2>
+        <p class="content">I can help you build a product , feature or website Look through
+             some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.<br>
+        </p>
+        <div class="content">
+            <ul class="">
+                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-github" src="images/social/github-v.png" alt="github"></a></li>
+                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-github" src="images/social/Linkedin.png" alt="github"></a></li>
+                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-thumb" src="images/social/Thumb-Enabled.png" alt="thumb"></a></li>
+                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-twitter" src="images/social/Twitter.png" alt="twitter"></a></li>   
+                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-M" src="images/social/M-Disabled.png" alt="M"></a></li>   
+                </ul>
+        </div>
```

Comment:
- [ ] In keeping with [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) please fix your indentation as it makes your code neater.